### PR TITLE
add appveyor-retry command

### PR DESCRIPTION
**Pull request by mehagar**
See appveyor/ci#418 for how script works. The idea to use it came from https://help.appveyor.com/discussions/problems/3147-error-eperm-operation-not-permitted-rename-on-npm-install (See the last comment).

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ install:
   - ps: Install-Product node 8
   - chocolatey install make
   - chocolatey install gnuwin32-coreutils.portable
-  - npm install
+  - appveyor-retry npm install
```
Comment (sanmut) [inline .appveyor.yml]: This is great. Good find. 
Can (should) we set the number of retries at all?

File: .appveyor.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ install:
   - ps: Install-Product node 8
   - chocolatey install make
   - chocolatey install gnuwin32-coreutils.portable
```
Comment (rajsite) [inline .appveyor.yml]: Can you add the retry to the two above `chocolatey install` lines as well? They also fail appveyor builds occasionally.

File: .appveyor.yml (hunk 1 above)
Comment (mehagar) [reply .appveyor.yml]: I don't think so.

File: .appveyor.yml (hunk 2 above)
Comment (mehagar) [reply .appveyor.yml]: Done.

**Pull request merged**